.github/workflows/release.yml

```
@@ -2,8 +2,7 @@ name: Release
 
 on:
   push:
-    tags:
-      - 'v*' # Push events to matching v*, i.e. v1.0, v2.1.3
+    tags: '*'
```

I would prefer some way of distinguishing that this is really a version tag.

```suggestion
    tags: '*.*.*'
```

Something like that?

---

Sounds good!
Do you want to be even be more specific?

```suggestion
    tags: '[0-9]+.[0-9]+.[0-9]+'
```

Reference: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet